add missing ASN.1 Primitives (#257)
a-sit-plus/signum: CHANGELOG.md

## 3.0

### NEXT
* **Note: We are deprecating and will soon be removing the debug-only serialization for cryptographic datatypes like certificates, public keys, etc.**
    * We support robust ASN.1 encoding and mapping from/to JOSE and COSE datatypes and our ASN.1 structures support pretty printing.
    * -> There is no need for this misleading serialization support for debugging anymore
    * `@Serializable` suggests deserialization from JSON, CBOR, etc. works, which was never universally true
    * Getting native ASN.1 serialization for kotlinx-serialization is now a no-brainer given we support every primitive required.
    * This note will be prepended to the changelog entries until the `@Serialization` annotations have been remove.

>         * At the very latest, this will be Indispensable 4.0.0 / Supreme 1.0.0.
>         * Probably, this will happen earlier in reality.

Reviewer: ```suggestion
        * This will happen by Indispensable 4.0.0 / Supreme 1.0.0, if not before then.
```
